app/javascript/src/components/Dashboard/Notes/NotesCard/CardBody.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const CardBody = ({ description }) => {
+  return (
+    <div className="px-4 pt-2 text-gray">
+      <p className="pr-1 text-sm">{description}</p>
```

Comment:
Could use `Typography` component here. 
Also, wouldn't using `children` props here be better for a reusable component? This adds more flexibility.